cmd/mt-index-migrate/main.go

```diff
@@ -22,6 +22,8 @@ var (
 	dstCassAddr     = flag.String("dst-cass-addr", "localhost", "Address of cassandra host to migrate to.")
 	srcKeyspace     = flag.String("src-keyspace", "raintank", "Cassandra keyspace in use on source.")
 	dstKeyspace     = flag.String("dst-keyspace", "raintank", "Cassandra keyspace in use on destination.")
+	srcIdxTable     = flag.String("src-idx-table", "metric_idx", "Cassandra index table name in use on source.")
+	dstIdxTable     = flag.String("dst-idx-table", "metric_idx", "Cassandra index table name in use on destination.")
```

Comment:
No need for 'idx' . It is implied because it's an index tool. The other flags don't specify it either

Reply:
Thanks. Fixed in new commit.